projects/aca-content/src/lib/components/create-menu/create-menu.component.html

```diff
@@ -1,36 +0,0 @@
-<ng-container *ngIf="expanded">
-  <button
-    mat-stroked-button
-    data-automation-id="create-button"
-    [matMenuTriggerFor]="rootMenu"
-    title="{{ 'APP.NEW_MENU.TOOLTIP' | translate }}"
```

Comment:
We can probably remove this from translation files? 
<img width="998" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/ca2f0733-86ae-46be-ae07-a9470fa5373e">

Reply:
removed, thanks